module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -567,7 +607,235 @@ protected function getSupportedCitationFormats()
+     * @return string
+     */
+    public function getContainerIssue()
+    {
+        $numbering = $this->prioritizedFields('ContainerIssue');
+        if (is_array($numbering)) {
+            foreach ($numbering as $key => $data) {
+                if (isset($data['Type']) && strtolower($data['Type']) == 'issue') {
+                    return isset($data['Value']) ? $data['Value'] : '';
+                }
+            }
+        }
+        return '';
+    }
+    /**
+     * Get volume of containing record
+     * 
+     * @return string
+     */
+    public function getContainerVolume()
+    {
+        $numbering = $this->prioritizedFields('ContainerVolume');
+        if (is_array($numbering)) {
+            foreach ($numbering as $key => $data) {
+                if (isset($data['Type']) && strtolower($data['Type']) == 'volume') {
+                    return isset($data['Value']) ? $data['Value'] : '';
+                }
+            }
+        }
+        return '';
+    }
+    
+    /**
+     * Get ISSNs (of containing record)
+     * 
+     * @return array
+     */
+    public function getISSNs()
+    {
+        $issns = parent::getIssns();
+       
+        $identifiers = $this->prioritizedFields('ISSNs');
+        if (is_array($identifiers)) {
+            
+            foreach ($identifiers as $key => $data) {
+                
+                $type = strtolower($data['Type']);
+                if (isset($data['Type']) && isset($data['Value'])
+                    && ($type == 'issn-print' || $type == 'issn-electronic')
+                ) {
+                    $issns[] = $data['Value'];
+                }
+            }
+        }
+        return $issns;
+    }
+    
+    /**
+     * Get an array of ISBNs
+     * 
+     * @return array
+     */
+    public function getISBNs()
+    {
+        $isbns = parent::getISBNs();
+       
+        $identifiers = $this->prioritizedFields('ISBNs');
+        if (is_array($identifiers)) {
+            
+            foreach ($identifiers as $key => $data) {
+                
+                $type = strtolower($data['Type']);
+                if (isset($data['Type']) && isset($data['Value'])
+                    && ($type == 'isbn-print' || $type == 'isbn-electronic')
+                ) {
+                    $isbns[] = $data['Value'];
+                }
+            }
+        }
+        return $isbns;
+        
     }
+        
+    /**
+     * Get year of containing record
+     * 
+     * @return string
+     */
+    public function getContainerYear()
+    {
+        return $this->prioritizedFields('ContainerYear');
+    }
+    
+    /**
+     * Get year of containing record
+     * 
+     * @return string
+     */
+    public function getContainerPages()
+    {
+        $pages = '';
+        $pagination = $this->prioritizedFields('ContainerPages');
+        if (isset($pagination['StartPage'])) {
+            $pages = $pagination['StartPage'];
+        }
+        // TODO add end page parsing
```

Comment:
Sometimes, there is only a StartPage and an PageCount item. It might also be good to use a regex like https://regex101.com/r/9pN1bA/1 to deal with different spellings.